sdk/trace/benchmark_test.go

```diff
@@ -144,6 +145,7 @@ func BenchmarkSpanWithAttributes_all_2x(b *testing.B) {
 }
 
 func BenchmarkTraceID_DotString(b *testing.B) {
+	getTracer(b, "Benchmark traceID to string")
```

Comment:
Is this needed?

Reply:
It just doesn't look like the tracer is used in this benchmark. No big deal.

Reply:
I was looking in the wrong place. Yes, it is not needed. I have removed it.